Cargo.toml

```diff
@@ -43,6 +46,7 @@ reqwest = { version = "0.12.3", features = ["json"] }
 schemars = "=0.8.5"
 serde = { version = "1.0.193", default-features = false, features = ["derive"] }
 serde_json = { version = "1.0.108", features = ["preserve_order"] }
+tar = "0.4.40"
```

Comment:
Is it possible to remove the default feature and still work ? thanks

Reply:
Yes, it is. We don't need xattrs.